### PR TITLE
CkLocMgr: Refactor Pre-Cleanup using clang-format

### PR DESCRIPTION
**Pull request by epmikida**
Some initial cleanup for the CkLocMgr refactor. Re-formats cklocation files with clang-format, and removes deprecated code that got in the way.

### CODE REVIEW

**Comment by epmikida** [comment]
Also note that this is based on https://github.com/UIUC-PPL/charm/pull/3195, because the upcoming location manager changes are all affected by the barrier cleanup.

**Review by rbuch — DISMISSED** [review]

**Review by stwhite91 — DISMISSED** [review]

File: src/ck-core/ckmigratable.h [hunk 1]
```diff
@@ -123,14 +123,6 @@ class CkMigratable : public Chare {
...
   bool isInCore; //If true, the object is present in memory
 #endif
-
-#if CMK_FAULT_EVAC
-private:
-  bool asyncEvacuate;
-  void AsyncEvacuate(bool set){myRec->AsyncEvacuate(set);asyncEvacuate = set;};
-public:
-  bool isAsyncEvacuate(){return asyncEvacuate;};
-#endif
```
Comment (evan-charmworks) [inline src/ck-core/ckmigratable.h:133]: Would the removal of the AsyncEvacuate method in this PR break compilation with `-DCMK_FAULT_EVAC=1`? TCharm and AMPI both call `this->AsyncEvacuate` from chare arrays.

Comment (epmikida) [reply src/ck-core/ckmigratable.h:133]: I was under the impression that FAULT_EVAC is no longer being supported.

Comment (evan-charmworks) [reply src/ck-core/ckmigratable.h:133]: If it's not then all CMK_FAULT_EVAC blocks should be removed in a separate PR preceding this one. We should discuss this in Core today.

**Comment by evan-charmworks** [comment]
If this is just automated reformatting, could you update the PR title to say that specifically?

**Comment by epmikida** [comment]
> If this is just automated reformatting, could you update the PR title to say that specifically?

Done

**Comment by evan-charmworks** [comment]
Needs conflicts to be resolved

**Comment by epmikida** [comment]
> Needs conflicts to be resolved

Rebased on current master

**Review by evan-charmworks — APPROVED** [review]

**Review by rbuch — APPROVED** [review]

**Pull request merged**